pom.xml

```diff
@@ -55,6 +55,15 @@
             <groupId>org.rogach</groupId>
             <artifactId>scallop_2.12</artifactId>
         </dependency>
+        <!-- apache commons -->
+        <dependency>
+            <groupId>org.apache.commons</groupId>
+            <artifactId>commons-csv</artifactId>
+        </dependency>
+        <dependency>
```

Comment:
now that you use better-files, do you still require this dependency? If not, please remove

Reply:
This is still needed in method outputCurationList

Reply:
sorry, I meant `commons-io`, not `commons-csv` 😜 
See https://github.com/vesaakerman/easy-manage-curation-work/pull/1